src/control/SConscript

```diff
@@ -71,8 +72,11 @@ def install_go_bin(denv, gosrc, libs, name, install_name):
     install_bin = get_install_bin_path(denv, install_name)
     build_bin = join('$BUILD_DIR', 'src/control/bin', name)
     src = [mod_src]
-    if libs is not None:
-        src.extend(libs)
+
+    if libs is None:
+        libs = []
+    libs.extend(gurt_lib)
```

Comment:
(pylint-undefined-variable) Undefined variable 'gurt_lib'

Reply:
See my comment about for why we're seeing this, it's an artifact or our pylint/scons support.